sass/_guardians.scss

```diff
@@ -0,0 +1,24 @@
+.guardian {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+
+    .guardian-picture {
+        width: 128px;
+        height: 128px;
+        min-width: 128px;
+        min-height: 128px;
+        border-radius: 128px;
+        margin: 20px;
+    }
+
+    .guardian-details {
```

Comment:
This has a unique name of its own, it probably shouldn't be nested inside `.guardian {}`.